test/acceptance/test/testsuite_common_test.go

```diff
@@ -34,7 +33,7 @@ func TestAcceptance(t *testing.T) {
 	}
 
 	RegisterFailHandler(Fail)
-	gomega.RegisterFailHandler(GomegaFail)
+	//gomega.RegisterFailHandler(GomegaFail)
```

Comment:
Is this intended to still be commented out?

Reply:
nice catch, thanks